scripts/setup-wasm.mjs

```diff
@@ -1,23 +1,23 @@
 import path from 'path'
-import { readFile, writeFile } from 'fs/promises'
-import { copy, pathExists } from 'fs-extra'
+import fs from 'fs'
```

Comment:
Similar question here - can we use `fs/promises` instead?

Reply:
https://github.com/vercel/next.js/pull/56917#discussion_r1362421015